actions/auth0-variables.js

```diff
@@ -0,0 +1,6 @@
+export const AUTH_CONFIG = {
+  domain: 'gww.auth0.com',
+  clientId: 'OIzT7gmLYcxbLZWGDz7LAsX6i2iCP2tc',
+  callbackUrl: 'http://localhost:8000/loading',
+  audience: 'https://gww.auth0.com/api/v2/'
+};
```

Comment:
this should not be in version control since it is gww specific

Reply:
check